library/std/src/primitive_docs.rs

```diff
@@ -468,8 +468,8 @@ mod prim_unit {}
 ///
 /// [`null`]: ptr::null
 /// [`null_mut`]: ptr::null_mut
-/// [`is_null`]: ../std/primitive.pointer.html#method.is_null
-/// [`offset`]: ../std/primitive.pointer.html#method.offset
+/// [`is_null`]: pointer::is_null
+/// [`offset`]: pointer::offset
```

Comment:
This is not related to this PR specifically nor is it blocking, but should we recommend using things like () and ! more when possible ?

There are hundreds of links where a () could be added, maybe it's noise ?

Reply:
I definitely don't think we need to recommend `()` in the link address itself, it will never show up in the rendered documentation. I'm ambivalent about whether to show it in the link text; it doesn't *hurt*, but I don't know if it helps all that much.